Solutions/Corvus.Monitoring.AspnetCore.Mvc.Specs/Features/ControllerOperations.feature

```diff
@@ -0,0 +1,15 @@
+﻿Feature: Controller Operations
+
+As a developer
+When I add the ObservableActionMethodsAttribute to my controller
+And implement the IHaveObservableActionMethods interface
+I want my controller actions to have spans created for them automatically
+
```

Comment:
If the use of `[ObservableActionMethods]` requires an implementation of `IHaveObservableActionMethods`, should we also have a test that verifies that it reports the failure to do this? Or is that a PITA to write a test for?

Reply:
It's pretty easy to write a test for, but the use of `ObservableActionMethods` doesn't mandate implementing the interface - it's an optional extra.

Reply:
And this isn't relevant any more anyway.